pkg/volume/csi/csi_attacher_test.go

```diff
@@ -928,17 +928,19 @@ func TestAttacherVolumesAreAttachedWithInline(t *testing.T) {
 
 func TestAttacherDetach(t *testing.T) {
 	nodeName := "fakeNode"
+	resyncPeriod := 10 * time.Second
```

Comment:
Can you comment what this change to the resyncPeriod does? I don't quite see how this exercises your change (if indeed it's possible to at all)?

Reply:
c.plugin.volumeAttachmentLister.Get(attachID) can not get va from informer before this change,  after sleep CsiResyncPeriod(1 min), it can get va from informer. so i reduce the sleep time from 1 min to 10s.

i don't know why can not get va from informer after we create the va in the test.